rpm/patch.spec

```diff
@@ -2,11 +2,10 @@ Name:       patch
 Summary:    The GNU patch command, for modifying/upgrading files
 Version:    2.7.6
 Release:    1
-Group:      Development/Tools
 License:    GPLv3
-URL:        http://www.gnu.org/software/patch/patch.html
+URL:        https://github.com/sailfishos/patch
 Source0:    ftp://ftp.gnu.org/gnu/patch/patch-%{version}.tar.xz
```

Comment:
Extra point from changing this to %{name}-%{version}.tar.xz

Reply:
zstd might also work

Reply:
Sure.